src/gui/footer/footer_items_heaters.cpp

```diff
@@ -27,11 +36,11 @@ string_view_utf8 FooterItemHeater::static_makeViewIntoBuff(int value, std::array
     const uint current = std::clamp(int(temps.current), 0, 999);
     const uint target_or_display = std::clamp(int(temps.target_or_display), 0, 999);
 
-    int printed_chars = snprintf(buff.data(), buff.size(), left_aligned ? left_aligned_str : const_size_str, current, target_or_display);
+    int printed_chars = snprintf(buff.data(), buff.size(), draw_type == footer::ItemDrawType::Static ? const_size_str : left_aligned_str, current, target_or_display);
```

Comment:
Be very careful about such a construct. Please close the ternary operator into the parenthesis. I don't even think that it works as you expect.

Reply:
it does what I meant, but I agree parenthesis are more readable